### PR TITLE
minor grammar fixes to step-6 tutorial doc

### PR DESCRIPTION
**Pull request by krishnakumarg1984**

### CODE REVIEW

File: examples/step-6/step-6.cc [hunk 1]
```diff
@@ -321,9 +321,9 @@ void Step6<dim>::assemble_system()
   // constraints. The constrained nodes are still in the linear system (there
   // is a nonzero entry, chosen in a way that the matrix is well conditioned,
   // on the diagonal of the matrix and all other entries for this line are set
-  // to zero) but the computed values are invalid (i.e., the correspond entry
-  // in <code>system_rhs</code> is currently meaningless). We compute the
-  // correct values for these nodes at the end of the <code>solve</code>
+  // to zero) but the computed values are invalid (i.e., the corresponding
+  // entries in <code>system_rhs</code> is currently meaningless). We compute
```
Comment (drwells) [inline examples/step-6/step-6.cc]:
```suggestion
  // entries in <code>system_rhs</code> are currently meaningless). We compute
```

**Comment by tjhei** [comment]
I don't feel strongly about it, but I thought the sentence is correct as singular, because it refers to the single entry in the RHS in the constrained line described before.

**Comment by drwells** [comment]
I read it as referring to the nodes, which are plural.

**Comment by krishnakumarg1984** [comment]
Alright. Sorry about the confusion. We can discard this PR. 

Anyway, the word `correspond` sounds a bit odd grammatically, no?

**Pull request merged**